strawberryfields/ops.py

```diff
@@ -757,13 +757,24 @@ class MeasureFock(Measurement):
 
     ns = None
 
-    def __init__(self, select=None):
+    def __init__(self, dark_counts=None, select=None):
+        self.dark_counts = dark_counts
+
         if select is not None and not isinstance(select, Sequence):
             select = [select]
+
         super().__init__([], select)
 
     def _apply(self, reg, backend, shots=1, **kwargs):
-        return backend.measure_fock(reg, shots=shots, select=self.select, **kwargs)
+        samples = backend.measure_fock(reg, shots=shots, select=self.select, **kwargs)
```

Comment:
wonder if the additions can be vectorized? Other than that this looks good!

Reply:
Not sure how this could be vectorised. Do you mean the list comprehension below?

Reply:
are the samples a list or a numpy array?